src/content/blog/astro-570.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: 'Astro 5.7'
+description: 'Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.'
+homepageLink:
+  title: 'Astro 5.7'
+  subtitle: 'Available now!'
+publishDate: '2025-04-15'
+authors:
+  - matt
+  - florian
+  - ema
+  - nate
+coverImage: '/src/content/blog/_images/astro-570/blog-post-5-7.webp'
+socialImage: '/src/content/blog/_images/astro-570/og-astro-5-7.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import Note from '~/components/Note.astro';
+import Mention from '/src/components/Mention.astro';
+
+**Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.**
```

Comment:
nit: these are two words
```suggestion
**Astro 5.7 has a basket load of treats, including stable Sessions and SVG components and a new Experimental Fonts API.**
```

Reply:
I'm pretty sure it's fine to keep as a compound word

Reply:
I had way too many tabs open trying to verify one way or the other. Confirmed that *basketload* is one word, so we can use that as our model here and stick to one word.